config/application.rb

```diff
@@ -42,5 +42,6 @@ class Application < Rails::Application
     config.autoload_paths += %w(lib app/mailers/concerns)
     config.assets.paths << Rails.root.join('vendor', 'assets', 'moj.slot-picker', 'dist', 'stylesheets')
     config.middleware.delete 'ActiveRecord::QueryCache'
+    config.session_expires_after = 20.minutes
```

Comment:
I think we should rename this to `sessions_expire_after` (no s) so that it's consistent with the cookie parameter (`:expires_after`).